packages/compass-web/src/index.tsx

```diff
@@ -245,7 +245,15 @@ const CompassWeb = ({
     const connectionsManagerCurrent = connectionsManager.current;
     void (async () => {
       try {
-        await connectionsManagerCurrent.connect(connectionInfo);
+        await connectionsManagerCurrent.connect(connectionInfo, {
+          forceConnectionOptions: [],
+          onDatabaseSecretsChange() {
+            // noop
+          },
+          onNotifyOIDCDeviceFlow() {
+            // noop
+          },
```

Comment:
This should be optional, we shouldn't be required to pass it when it's not needed